cli-output/src/cli_output.rs

```diff
@@ -610,25 +610,56 @@ fn show_votes_and_credits(
         return Ok(());
     }
 
-    writeln!(f, "Recent Votes:")?;
-    for vote in votes {
-        writeln!(f, "- slot: {}", vote.slot)?;
-        writeln!(f, "  confirmation count: {}", vote.confirmation_count)?;
```

Comment:
Condensed this two lines to one for the precious vertical screen estate. :)

Reply:
NIT: What do you think about saving the per-line field name redundancy with a header line at the top?

Reply:
Hmm, I actually like this `epoch-info`-style (`key: value`). I might do this once I got enough motivation in the future. :)